content/en/docs/reference/config/security/normalization/index.md

```diff
@@ -13,7 +13,7 @@ For more information, please refer to [authorization normalization best practice
 
 ## Path Related
 
-This applies to the `paths` and `notPaths` field.
+This applies to the [`paths`](/docs/reference/config/security/authorization-policy/#Operation-paths) and [`notPaths`](/docs/reference/config/security/authorization-policy/#Operation-not_paths) field.
```

Comment:
I tried moving field into the link to something is highlighted blue and leaves the back-ticked field name in red.
```suggestion
This applies to the [`paths` field](/docs/reference/config/security/authorization-policy/#Operation-paths) and the [`notPaths` field](/docs/reference/config/security/authorization-policy/#Operation-not_paths).
```
<img width="424" alt="Screenshot 2024-04-25 at 9 32 30 AM" src="https://github.com/istio/istio.io/assets/10537847/0b3f807c-5493-4b74-8cd5-1daa33fca3d6">